astropy/nddata/bitmask.py

```diff
@@ -347,10 +596,14 @@ def bitfield_to_boolean_mask(bitfield, ignore_flags=0, flip_bits=None,
 
     Examples
     --------
+
         >>> from astropy.nddata import bitmask
         >>> import numpy as np
         >>> dqbits = np.asarray([[0, 0, 1, 2, 0, 8, 12, 0],
```

Comment:
Is dqbits a good name for this quantity? I would favor dqvals or something like that